portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,34 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+function addRandomFact(){
+    const facts = 
+    ['Hannah has tried to learn arabic 10 times. She still does not know it.',
+    'Hannah was once a girl scout!',
+    'Hannah has broken 1 bone and it from falling off a swing set.',
+    'Hannah has 1 sibling: a younger sister named Emily.',
+    'Hannah has played almost every pokemon game up onto USUM.',
+    'Hannah can not get goosebumps and is not quite sure why not.',
+    'Hannah\'s go to starbucks drink is a Mocha Latte',
+    'At one point, Hannah had 6 pets at once (no fish!)',
+    'Hannah\'s current favorite music genre is indie',
+    'Hannah\'s dream vacation is a trip to Peru.',
+    'Hannah spends 95% of her time on Tiktok (hamanthasalami).',
+    'Hannah is genetically predispositioned to have more bug bites than average.',
+    'Hannah is a fan of 100 gecs.',
+    'Hannah\'s first concert was Beyonce.',
+    'Hannah was NOT an ugly baby, but she was an ugly middle schooler.',
+    'Hannah has 10 fingers and 10 toes.',
+    'Hannah can trace her ancestory back to 1521.',
+    'Hannah has been to more countries than states (despite living here her whole life).',
+    'At one point, Hannah spoke passable Mandarin. That point is not now, sadly.',
+    'Hannah thinks she\'s funny, but thats debatable'
+    ];
+    const fact = facts[Math.floor(Math.random()*facts.length)];
+    const factContainer = document.getElementById('fact-container');
+    factContainer.innerHTML = "";
+    var tag = document.createElement("p");
+    var factTxt = document.createTextNode(fact);
```

Comment:
factTxt is a bit misleading. Given that we have fact already, I can assume that this is different than just a string representation of a fact. But if I didn't know that, i would assume that this is just a string wrapped in quotes. It might be useful to call this factTextNode to clear up any ambiguity.

Reply:
changed!